libs/hdf-converters/data/converters/cciListXml2json.ts

```diff
@@ -2,28 +2,31 @@ import fs from 'fs';
 import * as _ from 'lodash';
 import xml2js from 'xml2js';
 
+// Documentation is located at https://github.com/mitre/heimdall2/wiki/Control-Correlation-Identifier-(CCI)-Converter.
 const parser = new xml2js.Parser();
 const pathToInfile = process.argv[2];
```

Comment:
use an actual argparse library

https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig
http://yargs.js.org/
https://github.com/tj/commander.js

others are out there - I would try the built-in first

process.argv is enough to get off the ground with a one-off script but now that we're giving this a polish, might as well put in the polish